packages/datadog-instrumentations/src/fastify.js

```diff
@@ -142,6 +152,10 @@ function getRes (reply) {
   return reply && (reply.raw || reply.res || reply)
 }
 
+function getCodeOriginForSpansTags (request) {
+  return request?.routeOptions?.config?.[kCodeOriginForSpansTagsSym]
```

Comment:
if you do my other suggested changes, you can do this in the `apm:fastify:request:handle` handler, and have the `Symbol` defined in `packages/datadog-plugin-fastify/src/index.js`